libs/nx-esbuild/src/executors/build/executor.ts

```diff
@@ -10,9 +11,16 @@ export default async function runExecutor(
     if (!context.projectName) {
         throw new Error('No projectName')
     }
-    const libRoot = context.workspace.projects[context.projectName].root
```

Comment:
renamed to app for better semantics?

Reply:
Yeah, esbuild will be used for node apps, not libraries so they will be created in the app folder.

This was a copy paste from one of the library executors